code/analysis/analysis_civil/figures.do

```diff
@@ -661,10 +702,12 @@ if `event' == 1 {
 		xtitle("Years relative to treatment", size(medium))
 		ytitle("Effect of MeToo on win", size(medium))
 		`xlabel'
+		text(.45 2.1 "ATT (Female - Male): `att_did'", size(medsmall) color(black))
+		text(.55 1.6 "ATT (Male): `att'", size(medsmall) color(black))
```

Comment:
same comment as above, let's make it men and women rather than difference between